WTWSMS/decisions/VIET_culture_decisions.txt

```diff
@@ -198,15 +198,18 @@ decisions = {
 			age = 16
 			scaled_wealth = 0.5
 			prisoner = no
-			religion_group = christian
-			NOT = { has_character_flag = VIET_celtic_high_cross }
 			OR = {
-				culture = saxon
+				religion = insular
+				religion = pelagianism
 				AND = {
-					culture_group = celtic
-					NOT = { culture = breton }
+					religion_group = christian
+					OR = {
+						culture = saxon
+						culture = old_saxon
+					}
```

Comment:
You explicitly asked me to instate that. You told me that all Christian Saxons should be able to erect High Crosses, regardless of what Christian they were.

In addition, any Insular Christian can erect High Crosses, because that was a major facet of the religion.